draft-ietf-mls-protocol.md

```diff
@@ -2479,19 +2479,60 @@ defined in {{RFC5116}}. AEAD_CHACHA20_POLY1305 is defined
 in {{RFC7539}}. The corresponding hash algorithms are defined in {{!SHS}}.
 
 It is advisable to keep the number of ciphersuites low to increase the chances clients can interoperate in a federated environment, therefore the ciphersuites only inlcude modern, yet well-established algorithms.
-Depending on their requirements, clients can choose between two security levels (roughly 128 bit and 256bit). Within the security levels clients can choose between faster X25519/X448 curves and FIPS 140-2 compliant curves for Diffie-Hellman key negotiations. Additionally clients that run predominantly on mobile processors can choose ChaCha20Poly1305 over AES-GCM for performance reasons. Since ChaCha20Poly1305 is not listed by FIPS 140-2 it is not paired with FIPS 140-2 compliant curves. The security level of symmetric encryption algorithms and hash functions is paired with the security level of the curves.
+Depending on their requirements, clients can choose between two security levels (roughly 128-bit and 256-bit). Within the security levels clients can choose between faster X25519/X448 curves and FIPS 140-2 compliant curves for Diffie-Hellman key negotiations. Additionally clients that run predominantly on mobile processors can choose ChaCha20Poly1305 over AES-GCM for performance reasons. Since ChaCha20Poly1305 is not listed by FIPS 140-2 it is not paired with FIPS 140-2 compliant curves. The security level of symmetric encryption algorithms and hash functions is paired with the security level of the curves.
 
 The mandatory-to-implement ciphersuite for MLS 1.0 is
-`MLS10\_128\_HPKE25519\_AES128GCM\_SHA256\_Ed25519` which is using
+`MLS10\_128\_HPKE25519\_AES128GCM\_SHA256\_Ed25519` which uses
 Curve25519, HKDF over SHA2-256 and AES-128-GCM for HPKE,
 and AES-128-GCM with Ed25519 for symmetric encryption and
 signatures.
 
 Values with the first byte 255 (decimal) are reserved for Private Use.
 
-New cipher suite values are assigned by IANA as described in
+New ciphersuite values are assigned by IANA as described in
 {{iana-considerations}}.
 
+## MLS Designature Expert Pool {#de}
+
+[[ OPEN ISSUE: pick DE mailing address.
+Maybe mls-des@ or mls-de-pool. ]]
```

Comment:
Suggest `mls-ciphersuites@`

Reply:
But, we're going to end up with more than one registry (e.g., extensions) so I was going for the more general name.